angular-part/src/app/core/services/news.service.ts

```diff
@@ -27,16 +26,24 @@ export class NewsService {
   }
 
   public getUserNews(): Observable<NewsModel[]> {
-    return of(this._userNews);
+    return this.http.get((CONFIG.MY_NEWS_URL))
+      .pipe(
+        map((data: any) => {
```

Comment:
Please avoid "any" type. Ok, if you meant data from server, but you always known that is an array. It would be great, if you use 'Array<any>'